cmd/terraform/linter/main.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"errors"
```

Comment:
Can we use `pkg/errors`

Reply:
wasn't sure there was much point having stack traces for a cli but can do